docs/website/docs/user-guides/advanced/connecting-a-service.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Connecting to a service
+sidebar_position: 1
+---
+
+This tutorial will show you how you can connect your Go application to a mongodb service.
+
+Building on top of the [Go quickstart guide](../quickstart/go.md), we will extend the application to check if it is connected to a mongodb service.
+
+### Pre-requisites
+1. Install Service Binding Operator on your cluster.  See [installation instruction](https://operatorhub.io/operator/service-binding-operator).
```

Comment:
Explain the `installation instruction` part?

Reply:
If I do that, then I'll have to explain for the other operator as well. I think I'll let the user work it out.
But, I think it would be nice to add why this operator should be installed, no?